packages/searchbase/src/index.d.ts

```diff
@@ -107,6 +107,8 @@ export class SearchBase {
 
   dataField: string | Array<string | DataField>;
 
+  aggregationField: string;
```

Comment:
@ShahAnuj2610 It should be an optional prop, I think you need to fix the typings for others too.